decidim-core/lib/decidim/form_builder.rb

```diff
@@ -826,7 +826,7 @@ def selected_scopes(attribute)
     # Modified to change the tag to a valid HTML tag inside the <label> element.
     def error_and_help_text(attribute, options = {})
       html = ""
-      html += content_tag(:span, options[:help_text], class: "help-text") if options[:help_text]
+      html += content_tag(:div, options[:help_text], class: "help-text") if options[:help_text]
```

Comment:
> I dunno understand why [the core tests](https://github.com/decidim/decidim/runs/7762817791?check_suite_focus=true) are failing. They're referring to some view that it wasn't even edited. I thought it was flaky, but still returning error over and over.
> @ahukkanen can we skip it?

@Crashillo The problem can be found on this line. You are making the `help-text` elements `<div>`s which is problematic when we want to display them inside `<label>` elements as the error shows.

We should fix the issue and change the `:div` into `:span` here. After that, the spec should pass.

Reply:
This is also the same reason why the [admin unit tests](https://github.com/decidim/decidim/runs/7762816666?check_suite_focus=true) are failing.

Reply:
And the [admin system specs](https://github.com/decidim/decidim/runs/7762817245?check_suite_focus=true) seem to be failing because the format of the "required field" has slightly changed.

E.g. here:
https://github.com/decidim/decidim/blob/4b87e1c9e6a3b0533c08b428d871d6d0ccbad261/decidim-admin/spec/shared/manage_impersonations_examples.rb#L42

It should actually be:
```ruby
expect(page).to have_selector("label", text: "Document number*Required field\nis invalid")
```

Notice the removal of the `\n` character after the star character.